<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseTopicDialog</class>
 <widget class="QDialog" name="ChooseTopicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose message type</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="choose_proto_button">
      <property name="text">
       <string>Protobuf</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="choose_json_button">
      <property name="text">
       <string>JSON</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
